vyper/semantics/namespace.py

```diff
@@ -7,6 +7,7 @@
     StructureException,
     UndeclaredDefinition,
 )
+from vyper.utils import get_levenshtein_string
```

Comment:
I think we should a) rename this to `get_levenshetein_error_suggestions` and b) move all the levenshtein utils to their own module. i think `vyper/semantics/validation/levenshtein_utils.py` makes sense.